src/main/java/json_handler/JSONHandler.java

```diff
@@ -0,0 +1,102 @@
+package json_handler;
+
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+public class JSONHandler {
+    public static JSONObject parse(String jsonString) {
+        return new JSONObject(jsonString);
+    }
+
+    public static JSONArray parseArray(String jsonString) {
+        return new JSONArray(jsonString);
+    }
+
+    public static JSONObject run(String operation, String queryPath, JSONObject jsonObject) {
+        return run(operation, queryPath, "", jsonObject);
+    }
+
+    public static JSONObject run(String operation, String queryPath, String value, JSONObject jsonObject) {
+        JSONObject newObject = jsonObject;
+        switch (operation) {
+            case "replace":
+                newObject = replace(jsonObject, queryPath, value);
```

Comment:
Instead of assigning a new variable, potentially re-assigning it if it fallls inside a case statement, and then returning it, we could just return inside the case statement and add a default case:

```java
public static JSONObject run(String operation, String queryPath, String value, JSONObject jsonObject) {
        switch (operation) {
            case "replace":
                return replace(jsonObject, queryPath, value);
            case "remove":
                return remove(jsonObject, queryPath);
            case "add":
                return add(jsonObject, queryPath, value);
            default:
                return jsonObject;
        }
}
```